README.md

```diff
@@ -307,6 +319,32 @@ for page in pager:
 ```
 
 
+### Autocomplete
+
+OpenAlex reference: [Autocomplete entities](https://docs.openalex.org/how-to-use-the-api/get-lists-of-entities/autocomplete-entities).
+
+Autocomplete a string:
+```python
+from pyalex import Autocompletes
+
+Autocompletes()["stockholm"]
```

Comment:
```suggestion
from pyalex import autocomplete

autocomplete("stockholm")
```

My suggestion would be to rewrite it into this as autocomplete is not a separate resource (although the OpenAlex API design might let users think this). 

With this change, we reserve the item retrieval for identifiers.